src/pages/Settings.svelte

```diff
@@ -52,6 +52,13 @@
     localStorage.setItem("itemSkin", $preferences.itemSkin);
   };
 
+  const toggleHidden = (e) => {
+		e.preventDefault();
+		$preferences.showHidden = $preferences.showHidden == "hidden" ? "shown" : "hidden";
+
+		localStorage.setItem('showHidden', $preferences.showHidden);
```

Comment:
Testing out your deployment link and found a bug.  If you've already visited a section, and then change this setting it won't take effect.  To repro, login, goto mounts, then goto setting, change setting to show unobtained, then click back into mounts.  The page is unaffected.  If you refresh the page it picks up new settings and reloads proper.

This is because of us using a Cache inside the different apis to make things faster and reduce load.

Fixing this is a little tricky.  You'll need to invalidate the cache, what makes it a bit tricky is each api holds onto their own cache.  You don't really want to reach into each of them and reset it.  

My suggestion for one way to fix:
1) introduce a updated timestamp property on the Cache object, see _cache.js.  default being undefined.
2) inside the update method in the Cache class, set the timestamp to Date.now()
3) inside Settings, in method toggleHidden(), add another item to storage, say showHiddenUpdated, and set that value to Date.now()
4) add getShowHiddedUpdated to utils to read this item
5) finally, inside Cache, modify the isValid method to also compare this saved value against it's local updated timestamp.  If the one in local storage is newer than the cached value, then return false, and the api layers should recreate the cache.  Of course you'll need to also cover edge cases for when things don't exist in localstorage, etc.

Side note:  I'm not a big fan on where we've landed with this pattern for storage and util methods, etc.  Feels like this should probably be fronted by preferences fully, but you don't need to make that change now.  You copying the pattern is fine for consistency.